tests/storage/utils.py

```diff
@@ -213,6 +211,7 @@ def create_role_binding(
 @contextmanager
```

Comment:
Please add typing.

Reply:
Added for the functions that are related to this PR